r2r/cli/cli.py

```diff
@@ -72,10 +72,11 @@ def cli(ctx, config_path, config_name, client_server_mode, base_url):
 @cli.command()
 @click.option("--host", default="0.0.0.0", help="Host to run the server on")
 @click.option("--port", default=8000, help="Port to run the server on")
+@click.option("--reload", default=False, help="Reload the server on change")
 @click.pass_obj
-def serve(obj, host, port):
+def serve(obj, host, port, reload):
```

Comment:
if we are modifying this, can we support config_name as an input variable as we do in the quickstart?

Reply:
sure, will add that too